tools/ansible.md

```diff
@@ -0,0 +1,33 @@
+---
+permalink: /ansible
+layout: post
+title: Ansible
+command: ansible --version
+link: https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html
+changelogTemplate: https://github.com/ansible-community/ansible-build-data/blob/main/__RELEASE_CYCLE__/CHANGELOG-v__RELEASE_CYCLE__.rst
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+activeSupportColumn: true
+eolColumn: Supported
+iconSlug: ansible
+
+releases:
+  - releaseCycle: 4
+    release: 2021-07-20
+    eol: false
+    latest: "4.3.0"
+  - releaseCycle: 3
+    release: 2021-05-11
```

Comment:
```suggestion
    release: 2021-02-18
```

https://www.ansible.com/blog/announcing-the-community-ansible-3.0.0-package says Feb 18 as release date for v3.0.0

Reply:
I updated release dates regarding changelog https://github.com/ansible-community/ansible-build-data/blob/main/3/CHANGELOG-v3.rst#v3-0-0